src/components/ItemList/ItemListDumb.js

```diff
@@ -34,7 +34,7 @@ class ItemList extends Component {
               ) : (
                 items.map(item => <Item key={item.id} item={item} />)
               )}
-              <li className="list-item has-text-centered">Total: {total}</li>
+              <li className="list-item has-text-centered">Total: {total.toFixed(2)}</li>
```

Comment:
Nice way to format the monies...

Reply:
Not sure that I would want to replace it? I was making this router for logged in users and now I don't want that behavior. What would you recommend adding?

Reply:
OK good. I just didn't know why it was removed, with no context. Thanks for the clarification!